src/main/java/uk/gov/hmcts/darts/courtlogs/GetCourtLogsMapper.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.darts.courtlogs;
+
+import com.service.mojdarts.synapps.com.GetCourtLogResponse;
+import com.synapps.moj.dfs.response.CourtLog;
+import com.synapps.moj.dfs.response.CourtLogEntry;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.model.courtLogs.CourtLogs;
+import uk.gov.hmcts.darts.model.courtLogs.Entry;
+
+@Service
+public class GetCourtLogsMapper {
+
+    public GetCourtLogResponse toLegacyApi(CourtLogs courtLogs) {
+        var legacyCourtLog = new CourtLog();
+        legacyCourtLog.setCourthouse(courtLogs.getCourthouse());
+        legacyCourtLog.setCaseNumber(courtLogs.getCaseNumber());
+        courtLogs.getEntries()
+              .forEach((logEntry) -> legacyCourtLog.addCourtLogEntry(toLegacyApi(logEntry)));
+
+        var innerResponse = createInnerResponse();
+        innerResponse.setCourtLog(legacyCourtLog);
+        innerResponse.setCode("200");
+        innerResponse.setMessage("OK");
```

Comment:
Consider something like the following to avoid potential for mismatch of code and message if changes are made in the future:

```java
        var httpStatus = HttpStatus.OK;
        innerResponse.setCode(String.valueOf(httpStatus.value()));
        innerResponse.setMessage(httpStatus.name());
```